robots:
  arm:
    default_gripper: gripper

grippers:
  gripper:
    type: GenericGripper
    controller_ns: gripper_controller
    base_link: wrist_roll_link
    tip_link: gripper_link
    min_position: 0.0
    max_position: 0.1
    max_effort: 0.0

cameras:
  head_camera: DepthCamera

moveit_pose_reference_frame: base_link
moveit_eef_step: 0.05
